dnf-behave-tests/createrepo_c/steps/lib/file.py

```diff
@@ -83,7 +88,7 @@ def decompress_file_by_extension_to_dir(compressed_filepath, dest_dir):
 
     basename = os.path.basename(compressed_filepath)
     dst = os.path.join(dest_dir, basename)
-    if compressed_filepath.endswith(".bz2"):
+    if compressed_filepath.endswith(".bz2") or compressed_filepath.endswith(".zck"):
         dst = dst[:-4]
     else: # .gz and .xz are both 3 chars long
         dst = dst[:-3]
```

Comment:
Seems you could use https://docs.python.org/3/library/os.path.html#os.path.splitext for this...

Reply:
Thanks, that's nifty. :+1: